packages/fleet/fleet-agent/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,11 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -1,7 +1,7 @@
+ image:
+   os: "windows,linux"
+   repository: rancher/fleet-agent
+-  tag: v0.8.2-rc.3
++  tag: v0.8.1
```

Comment:
This doesn't look right to me. Also, this patch doesn't exist on the `dev-v2.7` branch.